extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -287,6 +318,11 @@ export class AmpAnalytics extends AMP.BaseElement {
           (this.config_['vars'] && this.config_['vars'][name]) || '');
       return val + argList;
     });
+
+    // Add any given extraUrlParams as query string params
+    if (this.config_['extraUrlParams']) {
+      request = addParamsToUrl(request, this.config_['extraUrlParams']);
```

Comment:
this assumes that the parameters are appended to the url using `&` and the key and value are separated by `=`. Since that is the standard way of doing it, I am fine with it. It is probably worth noting it in docs though.